src/app/tests/suites/certification/Test_TC_ALOGIN_1_12.yaml

```diff
@@ -50,13 +50,23 @@ tests:
               type: bitmap32
 
     - label: "Step 3: Read the global attribute: AttributeList"
+      PICS: PICS_EVENT_LIST_ENABLED
```

Comment:
The default could be to check list without, and IF PICS_EVENT_LIST_ENABLED have a single additional test item to validate eventlist (65530) is present.

Reply:
I considered asking for that, but that makes it harder in the future to switch to a "event list is always there" world.

On the other hand, we may never end up in that world to start with....

Reply:
This was an automated script ... you are applying a bit more human logic than the script has.

Reply:
Also for script inteligence - because default yaml did not preserve comments or indents, I ended up scripting line-text processing (ouch!) to edit things ... so list editing and other things is really really primitive (think string substitution).

There are things like ruamel which is supposed to try to preserve things, however I did not attempt to use it at this point.